Sources/Misc/DangerousSettings.swift

```diff
@@ -66,6 +66,12 @@ import Foundation
      */
     @objc public let customEntitlementComputation: Bool
 
+    /**
+     * Controls whether StoreKit 2 JWS tokens are sent to RevenueCat instead of StoreKit 1 receipts.
+     * Must be used in conjunction with the `usesStoreKit2IfAvailable configuration` option.
+     */
+    @objc public let usesStoreKit2JWS: Bool
```

Comment:
Since this is just a "dangerous" setting, even though technically it's a public API, I think we can get away with making this not a minor (so not using `feat`).

Reply:
Wondering what you have in mind for this: is this just for us to test, or do we expect others to be able to use this?
I'm asking because we _could_ make this `internal` only.

Reply:
Just for us but we need to be able to use it in PurchaseTester so `internal` would not work I think

Reply:
I would not put this in `DangerousSettings` then, let's move it to `InternalDangerousSettingsType`